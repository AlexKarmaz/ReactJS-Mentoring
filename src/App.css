.App {
    margin: 1rem;
    font-family: Arial, Helvetica, sans-serif;
}

body {
  padding: 0;
  margin: 0;
}

#root{
  display: flex;
  flex-direction: column;
}
