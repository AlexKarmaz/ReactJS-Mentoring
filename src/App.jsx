import React from "react";
import {hot} from "react-hot-loader";
import "./App.css";

class App extends React.Component{
  render(){
    return(
      <div className="App">
        <h1> Hello, World!!! </h1>
      </div>
    );
  }
}

export default hot(module)(App);