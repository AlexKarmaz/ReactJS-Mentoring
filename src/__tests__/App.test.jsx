import React from 'react';
import App from '../App.jsx';
import renderer from 'react-test-renderer';

test('App should render Hello World', () => {
  const component = renderer.create(
    <App/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
