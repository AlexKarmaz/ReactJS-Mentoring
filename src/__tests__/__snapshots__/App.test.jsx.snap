// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render Hello World 1`] = `
<div
  className="App"
>
  <h1>
     Hello, World!!! 
  </h1>
</div>
`;
